hydra-node/test/Hydra/Chain/Direct/Contract/Abort.hs

```diff
@@ -168,6 +169,12 @@ genAbortMutation (tx, _utxo) =
             [ ChangeInput input (replacePolicyIdWith testPolicyId otherHeadId output) (Just $ toScriptData Initial.ViaAbort)
             , ChangeMintedValue (removePTFromMintedValue output tx)
             ]
+    , SomeMutation IdenticalCommits <$> do
+        (_input, output, _d) <- elements healthyCommits
+        newInput <- arbitrary
+        -- XXX: Ideally we should need to modify the PT to simulate a proper new commit
+        -- FIXME: this shouldn't be green
+        pure $ AddInput newInput output
```

Comment:
@v0d1ch and me added this to highlight that this PR is not completely solving the gap. We need to make sure the commits are completely reimbursed, i.e. not two identical commits can be reimbursed by one output. A follow-up PR will address this (and the corresponding FIXME in the script)